seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SeataProperties.java

```diff
@@ -101,4 +105,13 @@ public SeataProperties setUseJdkProxy(boolean useJdkProxy) {
         this.useJdkProxy = useJdkProxy;
         return this;
     }
+
+    public String[] getExcludesForAutoProxying() {
```

Comment:
ing means doing？

Reply:
Yes, but sometimes it means gerund.Need to change to proxy?